pages/index.tsx

```diff
@@ -20,12 +21,77 @@ const Home: NextPage<HomeProps> = (props) => {
         azureMapsSubscriptionKey={
           process.env.NEXT_PUBLIC_AZURE_MAP_SUBSCRIPTION_KEY ?? ""
         }
-        geoJSON={props.geoJSON}
+        h3={props.h3}
       />
     </>
   );
 };
 
+const getH3Data = async (): Promise<HexagonData[]> => {
+  // First, we need to convert our lat long json file to H3!
+  const jsonLatLongFile = await fs
+    .readFile(
+      `${process.cwd()}/public/geojson/us-zip-code-latitude-and-longitude.json`,
+      "utf-8"
+    )
+    .then(JSON.parse);
+
+  const h3: HexagonData[] = [];
+  for (let i = 0; i < jsonLatLongFile.length; i++) {
```

Comment:
Styling nitpick -- when iterating over an array, prefer `for (const thing of arrayOfThings) {}`